docs/reference/adventureworks/armTemplates/auxiliary/policies.json

```diff
@@ -18903,6 +18903,555 @@
+                      "AmlCompute",
+                      "ComputeInstance"
+                    ]
+                  },
+                  {
+                    "field": "Microsoft.MachineLearningServices/workspaces/computes/vmSize",
+                    "notIn": "[[parameters('allowedVmSizes')]"
+                  }
+                ]
+              },
+              "then": {
+                "effect": "[[parameters('effect')]"
+              }
+            }
+          },
+          "name": "Deny-MachineLearningCompute-VmSize"
+        },
+        {
+          "properties": {
+            "displayName": "Deny-MachineLearningComputeCluster-RemoteLoginPortPublicAccess",
+            "mode": "Indexed",
+            "description": "Deny public access of clusters via SSH.",
+            "metadata": {
+              "version": "1.0.0",
+              "category": "Machine Learning"
+            },
+            "parameters": {
+              "effect": {
+                "type": "String",
+                "metadata": {
+                  "displayName": "Effect",
+                  "description": "Enable or disable the execution of the policy"
+                },
+                "allowedValues": [
+                  "Audit",
+                  "Disabled",
+                  "Deny"
+                ],
+                "defaultValue": "Deny"
+              }
+            },
+            "policyRule": {
+              "if": {
+                "allOf": [
+                  {
+                    "field": "type",
+                    "equals": "Microsoft.MachineLearningServices/workspaces/computes"
+                  },
+                  {
+                    "field": "Microsoft.MachineLearningServices/workspaces/computes/computeType",
+                    "equals": "AmlCompute"
+                  },
+                  {
+                    "anyOf": [
+                      {
+                        "field": "Microsoft.MachineLearningServices/workspaces/computes/remoteLoginPortPublicAccess",
+                        "exists": false
+                      },
+                      {
+                        "field": "Microsoft.MachineLearningServices/workspaces/computes/remoteLoginPortPublicAccess",
+                        "notEquals": "Disabled"
+                      }
+                    ]
+                  }
+                ]
+              },
+              "then": {
+                "effect": "[[parameters('effect')]"
+              }
+            }
+          },
+          "name": "Deny-MachineLearningComputeCluster-RemoteLoginPortPublicAccess"
+        },
+        {
+          "properties": {
+            "displayName": "Deny-MachineLearningComputeCluster-Scale",
+            "policyType": "Custom",
+            "mode": "Indexed",
+            "description": "Enforce scale settings for machine learning compute clusters.",
+            "metadata": {
+              "version": "1.0.0",
+              "category": "Machine Learning"
```

Comment:
Is this policy primarily towards cost optimization/restrictions?

Reply:
That is correct. Should I make any changes?

Reply:
Category should be changed then IMO.

Reply:
What category doe you want me to use?

Reply:
Let's use 'Budget' which already exist

Reply:
Also, would be great to highlight this in the description, that this is about controlling cost/usage/utilization

Reply:
@krnese Shoould I do the same for `Deny-MachineLearningCompute-VmSize`? This is also very much cost related.

Reply:
DONE